pypika/dialects.py

```diff
@@ -274,7 +274,9 @@ def __init__(self, **kwargs: Any) -> None:
         super().__init__(dialect=Dialects.ORACLE, **kwargs)
 
     def get_sql(self, *args: Any, **kwargs: Any) -> str:
-        return super().get_sql(*args, groupby_alias=False, **kwargs)
+        kwargs.setdefault("quote_char", None)
```

Comment:
It's probably better to use `QUOTE_CHAR = None` like how `class SnowFlakeQueryBuilder` does it. Could you double check whether that results in the same desired behaviour?

Reply:
Works :)